course_catalog/serializers_test.py

```diff
@@ -71,3 +74,36 @@ def test_generic_foreign_key_serializer():
     serializer = ProgramItemSerializer(program_item)
     with pytest.raises(Exception):
         assert serializer.data.get("item").get("id") == course_topic.id
+
+
+def test_favorites_serializer():
+    """
+    Test that the favorite serializer generic foreign key works and also rejects unexpected classes
+    """
+    user = UserFactory()
+    course = CourseFactory()
+    bootcamp = BootcampFactory()
+    user_list = UserListFactory(author=user)
+    program = ProgramFactory()
+    course_topic = CourseTopicFactory()
+
+    favorite_item = FavoriteItem(user=user, item=course)
+    serializer = FavoriteItemSerializer(favorite_item)
+    assert serializer.data.get("content_data").get("id") == course.id
```

Comment:
Might as well compare the whole thing:
```python
assert serializer.data.get("content_data") == CourseSerializer(course).data
```

Ditto for the other favorites below.

Reply:
👍